src/conversion/model-to-view-converters.js

```diff
@@ -91,6 +90,35 @@ export function insertText() {
 	};
 }
 
+/**
+ * Function factory, creates a default model-to-view converter for node remove changes.
+ *
+ *		modelDispatcher.on( 'remove', remove() );
+ *
+ * @returns {Function} Remove event converter.
+ */
+export function remove() {
+	return ( evt, data, conversionApi ) => {
+		// Find view range start position by mapping model position at which the remove happened.
+		const viewStart = conversionApi.mapper.toViewPosition( data.position );
+
+		// Find view range end by providing to mapper view container and expected model offset.
+		// Get the offset from passed model position and passed length of removed content.
+		const viewContainer = conversionApi.mapper.toViewElement( data.position.parent );
+		const viewEnd = conversionApi.mapper._findPositionIn( viewContainer, data.position.offset + data.length );
```

Comment:
Instead of using private method, we should provide mapper API. Also it we skip custom callbacks we can get wrong position if this callbacks change offset.

Reply:
> Also it we skip custom callbacks we can get wrong position if this callbacks change offset.

This is a bit true but a bit not.

`remove` event is fired for every item one by one. This means that if given kind of element (like `listItem`) has custom mapping, it should also provide custom removing converter. It works like this now and I think we don't experience any bugs when it comes to removing lists?

So, I'd just remember about it and deal with it if it will prove to be troublesome.

Reply:
Still, using the private API looks bad for me.